### DISCUSSION TITLE
Simple, cheap rte_433 supported humidity sensor? Acurite? Which one?

### DISCUSSION THREAD

**Discussion by eigenphase** [merbanan/rtl_433#2334] [Q&A] [answered]
Did anyone successfully decode one of these Acurite temp/humidity sensors?

https://www.acurite.com/shop-all/weather-instruments/weather-sensors-and-parts/sensors/indoor-outdoor-temperature-humidity-sensor.html

If not, is there a good, cheap option that's not immediately a full blown weather station (or has fancy displays etc)? I really need it only for remote measurement in my crawl space ...

**Comment by zuckschwerdt** [comment, marked as answer]
Happy with my LaCrosse TX141TH Bv2 for that use-case (no display, robust, long battery live). There are some rebrands of them.
